test/parallel/test-eslint-crypto-check.js

```diff
@@ -12,21 +12,32 @@ new RuleTester().run('crypto-check', rule, {
   valid: [
     'foo',
     'crypto',
-    `
-      if (!common.hasCrypto) {
-        common.skip();
-      }
-      require('crypto');
+    `if (!common.hasCrypto) {
+      common.skip();
+    }
+    require("crypto");
     `
   ],
   invalid: [
     {
       code: 'require("crypto")',
-      errors: [{ message }]
+      errors: [{ message }],
+      output:
+      `if (!common.hasCrypto) {
+        common.skip("missing crypto");
+      }
+      require("crypto");
+      `
     },
     {
       code: 'if (common.foo) {} require("crypto")',
-      errors: [{ message }]
+      errors: [{ message }],
+      output:
+      `if (!common.hasCrypto) {
+        common.skip("missing crypto");
+      }
+      require("crypto");
```

Comment:
This can not be the expected output because common is not imported. But not only that, the first statement `if (common.foo) {}` was just removed? That is definitely not right.